File: examples/petstore-sdk/go/sdk/find_pets_by_status_param_status.go
package petstore

type FindPetsByStatusParamStatus string

const (
	FindPetsByStatusParamStatusAvailable FindPetsByStatusParamStatus = "available"
	FindPetsByStatusParamStatusPending FindPetsByStatusParamStatus = "pending"
	FindPetsByStatusParamStatusSold FindPetsByStatusParamStatus = "sold"
)

File: examples/petstore-sdk/kotlin/sdk/com/example/petstore/model/FindPetsByStatus_Param_Status.kt
package com.example.petstore.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
enum class FindPetsByStatus_Param_Status {
    @SerialName("available") Available,
    @SerialName("pending") Pending,
    @SerialName("sold") Sold,
}
